### ISSUE TITLE
Sequential testing 

### ISSUE THREAD

**Issue by Gabrielcidral1** [david26694/cluster-experiments#202]
Currently we can use the class NormalPowerAnalysis.mde_time_line to see the evolution of MDE by the time. But I understand that this help to decide the experiment duration as a fixed period.  I would like to see this line as a sequential testing, where you decide a max duration of a test, but it can stop if you pass some threshold level in terms of alpha. 

2 references:
- https://www.youtube.com/watch?v=4rWOx5fOJbg
- https://booking.ai/sequential-testing-at-booking-com-650954a569c7

**Comment by Gabrielcidral1** [comment]
exactly! We can also set as an input how "aggressive" this increase of alpha can be.So it can follow a function like this (@matheusbuniotto idea):


![image](https://github.com/user-attachments/assets/fc54cad0-ea77-4c72-bf48-dcbfd2a338e2)

**Comment by david26694** [comment]
very cool, do you have an idea of how should we implement it?
